src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts

```diff
@@ -54,15 +54,15 @@ export abstract class SimpleFindWidget extends Widget {
 	private _foundMatch: boolean = false;
 
 	constructor(
-		@IContextViewService private readonly _contextViewService: IContextViewService,
-		@IContextKeyService contextKeyService: IContextKeyService,
-		private readonly _state: FindReplaceState = new FindReplaceState(),
-		private readonly _options: IFindOptions,
+		readonly state: FindReplaceState = new FindReplaceState(),
+		readonly options: IFindOptions,
+		contextViewService: IContextViewService,
+		contextKeyService: IContextKeyService,
 		private readonly _keybindingService?: IKeybindingService
```

Comment:
I think we should be able to pass this in for `webviewFindWidget` too and make it mandatory